pkg/apprepo/apprepos_handler_test.go

```diff
@@ -435,9 +423,10 @@ func TestSecretForRequest(t *testing.T) {
 	}
 
 	testCases := []struct {
-		name    string
-		request appRepositoryRequestDetails
-		secret  *corev1.Secret
+		name             string
+		requestNamespace string
```

Comment:
is this `requestNamespace` never used?

Reply:
True - it was before this change (as the namespace was set in `secretForRequest` but is no longer. Thanks, it'll be updated in the next PR.